CONFIGURE.md

```diff
@@ -70,16 +70,93 @@ It's recommended to create your configuration in YAML format or configure the ag
 
+### Configuring encryption for flow configuration
+
+To encrypt flow configuration set the following property to true.
+
+    # in minifi.properties
+    nifi.flow.configuration.encrypt=true
+
+### Configuring additional sensitive properties
+
+It is possible to set a comma seperated list of encrypted configuration options beyond the default sensitive property list.
+
+    # in minifi.properties
+    nifi.sensitive.props.additional.keys=nifi.flow.configuration.file, nifi.rest.api.password
+
+### Backup previous flow configuration on flow update
+
+It is possible to backup the previous flow configuration file with `.bak` extension in case of a flow update (e.g. through C2 or controller socket protocol).
+
+    # in minifi.properties
+    nifi.flow.configuration.backup.on.update=true
+
+### Set number of flow threads
+
+The number of threads used by the flow scheduler can be set in the MiNiFi configuration. The default value is 5.
+
+    # in minifi.properties
+    nifi.flow.engine.threads=5
+
+### OnTrigger runtime alert
+
+MiNiFi writes warning logs in case a processor has been running for too long. The period for these alerts can be set in the configuration file with the default being 5 seconds.
+
+    # in minifi.properties
+    nifi.flow.engine.alert.period=5 sec
+
+### Event driven processor time slice
+
+The flow scheduler can be configured how much time it should allocate at maximum for event driven processors. The processor is triggered until it has work to do, but no more than the configured time slice. The default value is 500 milliseconds.
+
+    # in minifi.properties
+    nifi.flow.engine.event.driven.time.slice=500 millis
+
+### Administrative yield duration
+
+In case an uncaught exception is thrown while running a processor, the processor will yield for the configured administrative yield time. The default yield duration is 30 seconds.
+
+    # in minifi.properties
+    nifi.administrative.yield.duration=30 sec
+
+### Bored yield duration
+
+If a processor is triggered but has no work available, it will yield for the configured bored yield time. The default yield duration is 100 milliseconds.
+
+    # in minifi.properties
+    nifi.bored.yield.duration=100 millis
+
+### Graceful shutdown period
+
+It is possible to configure a graceful shutdown period, the period the flow controller will wait to unload the flow configuration and stop running processors.
+
+    # in minifi.properties
+    nifi.flowcontroller.graceful.shutdown.period=30 sec
+
+### FlowController drain timeout
+
+Timeout period for finishing processing of flow files in progress when shutting down flow controller. When not set we do not wait for flow files to finish processing.
+
+    # in minifi.properties
+    nifi.flowcontroller.drain.timeout=500 millis
+
 ### SiteToSite Security Configuration
 
-    in minifi.properties
+    # in minifi.properties
 
-    enable tls
+    # enable tls
```

Comment:
A similar change could be done on lines 162, 164, 167, 169 and 172

Reply:
Updated in 5cda81179ff848d1613d39888426a95b0409a017